src/plugins/advanced_settings/public/management_app/components/field/field.tsx

```diff
@@ -100,7 +100,7 @@ export class Field extends PureComponent<FieldProps> {
     if (type === 'image') {
       this.cancelChangeImage();
       return this.handleChange({
-        value: getEditableValue(type, defVal),
+        value: getEditableValue(type, defVal, defVal),
         changeImage: true,
```

Comment:
When I look at the `getEditableValue` implementation, I'm not sure to understand how this changes anything:

https://github.com/elastic/kibana/blob/3730dd0779ed8efd74aee88f57422781ec1ac122/src/plugins/advanced_settings/public/management_app/components/field/field.tsx#L51-L56

What is the actual value of `defValue` when you reset an image field?

Reply:
The difference is passing through `null` for `defVal` as opposed to `undefined`. It seems that setting the actual value to  `undefined` is a noop. I assumed this was the intended behaviour, but maybe this is an issue somewhere else?